src/main/java/seedu/duke/views/WeeklyScheduleView.java

```diff
@@ -104,21 +104,32 @@ public static void printRow(List<ArrayList<String>> hourSchedule, int timePeriod
                 if (weeklyTask.get(i).isEmpty()) {
                     printToJustify(columnWidth);
                 } else {
+
                     String currentTask = weeklyTask.get(i).get(0); //get 1st task
+                    String[] words = currentTask.split(" ");
                     if (currentTask.length() < columnWidth) {
                         print(currentTask);
                         printToJustify(columnWidth - currentTask.length());
                         weeklyTask.get(i).remove(0);
+                    } else if (words[0].length() > columnWidth) {
```

Comment:
I think this should try to follow happy path as well? so we catch the out of bounds at the start as well